apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -77,323 +77,266 @@ export function InstructorQuestionSettings({
+                        </button>
+                      `
+                    : ''
+                }
+                ${
+                  questionGHLink
+                    ? html`<a target="_blank" href="${questionGHLink}"> view on GitHub </a>`
+                    : ''
+                }
+                <input
+                  type="text"
+                  class="form-control"
+                  id="qid"
+                  name="qid"
+                  value="${resLocals.question.qid}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  This is a unique identifier for the question. (e.g., "addNumbers")
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="type">Type</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="type"
+                  name="type"
+                  value="${resLocals.question.type}"
+                  disabled
+                />
+                <small class="form-text text-muted"> This is the type of question. </small>
+              </div>
+              <div class="form-group">
+                <div>Topic</div>
+                <div name="topic">
+                  ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
+                    topic: resLocals.topic,
+                  })}
+                </div>
+              </div>
+              <hr />
+              <div class="form-group">
+                <div>Tags</div>
+                <div name="tags">
+                  ${renderEjs(__filename, "<%- include('../partials/tags') %>", {
+                    tags: resLocals.tags,
+                  })}
+                </div>
+              </div>
+              <hr />
+              <div class="form-group">
+                <div>
+                  Issues
+                  ${renderEjs(__filename, "<%- include('../partials/issueBadge') %>", {
+                    count: resLocals.open_issue_count,
+                    issueQid: resLocals.question.qid,
+                    suppressLink: resLocals.suppressLink,
+                    urlPrefix: resLocals.urlPrefix,
+                  })}
+                </div>
+              </div>
+              <hr />
+              <div class="form-group">
+                <div>Assessments</div>
+                <div>
+                  ${
+                    resLocals.assessments
                       ? renderEjs(__filename, "<%- include('../partials/assessments') %>", {
                           assessments: resLocals.assessments,
                           urlPrefix: resLocals.urlPrefix,
                         })
-                      : ''}
-                  </td>
-                </tr>
-                ${sharingEnabled
+                      : ''
+                  }
+                </div>
+              </div>
+              </form>
```

Comment:
The fact that these both end up at the same indentation level means that there's probably a mismatched tag somewhere above. See if you can find it; if not, I can take a look myself!

Reply:
Good catch, my `<form>` and subsequent `<div>` were backwards. Updated in [f9945bf](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/f9945bf0e4c4e0d20d0a34f07c433d9e135dc0c7).